code/__HELPERS/text.dm

```diff
@@ -256,14 +256,14 @@
 //Returns a string with reserved characters and spaces before the first letter removed
 /proc/trim_left(text)
 	for(var/i = 1 to length(text))
-		if(text2ascii(text, i) > 32)
+		if(text2ascii(text, i) != 32)
```

Comment:
This line (and the similar one in trim_right) is what I'm not quite sure of. Are such control characters used in cyrillic?

Reply:
It was changed to that in the oldpara220, so I'm not quite sure. Was mostly porting the cyrillic fix from the old host-coder

Reply:
But it doesn't seem to care about it, removed it